deployer/billing.py

```diff
@@ -0,0 +1,210 @@
+"""
++TODO: Validate billing export is setup for all clusters we 'manage'
++TODO: Write a JSON Schema for the billing source of truth
++TODO: Differentiate between billing accounts we should *invoice* and ones we don't need to *invoice*
++TODO: Write documentation on when to use this and who uses it
+"""
+import re
+from datetime import datetime
+from enum import Enum
+from pathlib import PosixPath
+
+import gspread
+import typer
+from dateutil.relativedelta import relativedelta
+from google.cloud import bigquery
+from rich.console import Console
+from rich.table import Table
+from ruamel.yaml import YAML
+
+from .cli_app import app
+from .file_acquisition import get_decrypted_file
+from .helm_upgrade_decision import get_all_cluster_yaml_files
+
+yaml = YAML(typ="safe")
+
+HERE = PosixPath(__file__).parent.parent
+
+
+def month_validate(month_str: str):
+    """
+    Validate passed string matches YYYY-MM format.
+
+    Returns values in YYYYMM format, which is used by bigquery
+    """
+    match = re.match(r"(\d\d\d\d)-(\d\d)", month_str)
+    if not match:
+        raise typer.BadParameter(f"{month_str} should be formatted as YYYY-MM")
```

Comment:
give an example should be formatted as YYYY-MM e.g. 2022-01

Reply:
Done